src/woo_publications/publications/models.py

```diff
@@ -50,6 +57,23 @@ class Document(models.Model):
         ),
         on_delete=models.CASCADE,
     )
+    document_service = models.ForeignKey(
+        "zgw_consumers.Service",
+        verbose_name=_("Documents API Service"),
+        on_delete=models.PROTECT,
+        limit_choices_to={
+            "api_type": APITypes.drc,
+        },
+        null=True,
+        blank=True,
+    )
+    document_uuid = models.UUIDField(
+        _("Document UUID"),
```

Comment:
```suggestion
        _("document UUID"),
```

and please add a help text along the lines of "The UUID of the API resource recorded in the Documenten API."